app/models/request.rb

```diff
@@ -110,6 +115,63 @@ def date_sort_key
     ]
   end
 
+  def circ_record
+    return unless cdl? && cdl_circ_record_key
+
+    @circ_record ||= begin
+      record = Checkout.find(cdl_circ_record_key, cdl: true)
+      return unless record.checkout_date == cdl_circ_record_checkout_date
+
+      record
+    end
+  end
+
+  def cdl?
```

Comment:
Can we order these based on the offset in the CDL comment?

Reply:
Reordered.